media-plugins/vdr-streamdev/vdr-streamdev-0.6.3.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR Plugin: Client/Server and http streaming plugin"
+HOMEPAGE="https://github.com/vdr-projects/vdr-plugin-streamdev"
+SRC_URI="https://github.com/vdr-projects/vdr-plugin-streamdev/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/vdr-plugin-streamdev-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="client +server"
+REQUIRED_USE="|| ( client server )"
+
+BDEPEND="acct-user/vdr"
+DEPEND=">=media-video/vdr-2.3"
+RDEPEND="${DEPEND}"
```

Comment:
I would put the acct-* deps in DEPEND that also gets inherited by RDEPEND. Both of them.